typescript/cli/src/commands/registry.ts

```diff
@@ -71,19 +72,34 @@ const listCommand: CommandModuleWithContext<{ type: ChainType }> = {
 /**
  * Addresses command
  */
-const addressesCommand: CommandModuleWithContext<{ name: string }> = {
+const addressesCommand: CommandModuleWithContext<{
+  name: string;
+  contract: string;
+}> = {
   command: 'addresses',
+  aliases: ['address', 'addy'],
```

Comment:
addy?

Reply:
lol this was h/t @tkporter

Reply:
I dont see a big downside to additional aliases so will resolve